### PR TITLE
test(storage): run cli cmds in docker-compose

### PR DESCRIPTION
**Pull request by artek-koltun**

### CODE REVIEW

**Review by intelfisz — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -79,3 +79,47 @@ Generate mocks like this:
 go install github.com/vektra/mockery/v2@latest
 make mock-generate
 ```
+
+## CLI
+
+Storage example
```
Comment (intelfisz) [inline README.md:85]: maybe level-3 heading?

Comment (artek-koltun) [reply README.md:85]: Then it would go under `Installation` chapter. Will it be correct?

Comment (intelfisz) [reply README.md:85]: I just meant a subsection of CLI section for storage examples. When we add other domains, they will be in their own subsections.

Comment (artek-koltun) [reply README.md]: Updated

File: README.md [hunk 2]
```diff
@@ -79,3 +79,47 @@ Generate mocks like this:
...
+
+# connect to local nvme/pcie ssd controller
+nvmf1=$(dpu storage create backend nvme controller --id nvmf1 --multipath disable)
+path1=$(dpu storage create backend nvme path pcie --controller "$nvmf1" --id path1 --bdf "0000:40:00.0")
+
+# expose volume over nvme/tcp controller
+ss0=$(dpu storage create frontend nvme subsystem --id subsys0 --nqn "nqn.2022-09.io.spdk:opitest0")
+ns0=$(dpu storage create frontend nvme namespace --id namespace0 --volume "Malloc0" --subsystem "$ss0")
+ctrl0=$(dpu storage create frontend nvme controller tcp --id ctrl0 --ip "127.0.0.1" --port 4420 --subsystem "$ss0")
+
```
Comment (glimchb) [inline README.md:102]: can you also add example for `get` command  ? I see create/delete only

Comment (artek-koltun) [reply README.md:102]: `get` is not added to cli set yet. When it is there, I will definitely add it to the docu

**Review by sandersms — APPROVED** [review]
Approved without get which will add when in the cli set

**Pull request merged**